node-wit/index.d.ts

```diff
@@ -0,0 +1,81 @@
+// Type definitions for node-wit 4.2
+// Project: https://github.com/wit-ai/node-wit#readme
+// Definitions by: Julien Dufresne <https://github.com/julienduf>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as Promise from "bluebird";
+
+export namespace log {
+
+    class Logger {
+
+        constructor(lvl: string);
+    }
+
+    const DEBUG: string;
+    const INFO: string;
+    const WARN: string;
+    const ERROR: string;
+}
+
+export interface WitEntityValue {
+
+    value?: string;
+    expressions?: string[];
+}
+
+export interface WitEntity {
+
+    id?: string;
+    values?: WitEntityValue[];
+}
+
+export interface WitContext {
+
+    state?: string[];
+    reference_time?: string;
+    timezone?: string;
+    entities?: WitEntity[];
+    location?: string;
+}
+
+export interface WitRequest {
+
+    sessinId?: string;
+    context?: WitContext;
+    text?: string;
+    entities?: WitEntity[];
+}
+
+export interface WitResponse {
```

Comment:
This interface (and others like `WitRequest`) aren't used anywhere - can you add tests for them, partially to demonstrate intended usage, but partially to add coverage?